main/webapp/modules/core/scripts/project/history-panel.html

```diff
@@ -1,7 +1,11 @@
 <div class="history-panel-help" bind="helpDiv">
   <h1 bind="or_proj_undo"></h1>
   <p bind="or_proj_mistakes"></p>
-  <p><a href="https://docs.openrefine.org/manual/running#history-undoredo" target="_blank"><b><span bind="or_proj_learnMore"></span></b></a></p>
+    <p><a href="https://docs.openrefine.org/manual/running#history-undoredo" target="_blank"><b><span bind="or_proj_learnMore"></span></b></a></p>
+    <div class="close-side-panel-options">
+      <label id="side-panel-checkbox-label" class="checkbox"></label>
+      <button class="close-btn" id="close-history-panel"></button>
+    </div>
```

Comment:
Here, I don't see a common naming logic.
`close-side-panel-options` -> `panel-splash-close` ? Because it's shared amoung all the panels of type splash…
`side-panel-checkbox-label` -> `history-panel-checkbox-label` ? Side is very generic. It's an history panel from the start.
`close-btn` -> `panel-splash-close-btn` ? There could be other close button. Maybe the generic name is shared and though better. Just a thought.
`close-history-panel` -> `history-panel-close` ? Same as above. This is an ID, it must point uniquelly.